scvi/core/models/__init__.py

```diff
@@ -32,16 +32,42 @@
 class VAEMixin:
```

Comment:
Global question. Is it normal this is in a __init__ file? I would expect that's not good practice. Should we have a specific file for these primitives instead?

Reply:
I've seen it before, but we should evaluate the organization of everything after all the huge changes are in place.